standards/rmrk2.0.0/entities/base.md

```diff
@@ -0,0 +1,200 @@
+# Base
+
+A base is a meta-entity, stored on-chain or off chain. It is not an NFT, but can be thought of as

```

Comment:
if this is stored off-chain, any guides on where and how to update it?

Reply:
Whether it's on of off chain does not matter does it? You always get a JSON and then consolidate updates to its fields on top?

Reply:
I mean more like IPFS, if we store base on IPFS then we would have to create new cid every time right? Maybe good to put that note somewhere? Or it's up to implementator to decide how to handle it?

Reply:
Would be up to the implementer like any other minting op in RMRK, but this does indeed complicate storage especially as a base has many IPFSed parts. Perhaps we can run some metrics on how much this would cost and check if it's worth fully storing on chain in RMRK 🤔